src/components/cylc/cylcObject/Menu.vue

```diff
@@ -43,6 +43,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             <v-icon :disabled=!authorised large>{{ mutation._icon }}</v-icon>
           </v-list-item-avatar>
           <v-list-item-content>
+            <!-- Is this safe against XSS scripting? -->
```

Comment:
Valid comment?
```suggestion
```

Reply:
Your comment?

Reply:
Automatically added by build process

Reply:
???

Reply:
When I ran `yarn install` it added this line.

Reply:
Will raise as an issue, may be able to do this better, otherwise delete.

Reply:
https://github.com/cylc/cylc-ui/issues/875